js/watermark.js

```diff
@@ -0,0 +1,54 @@
+function addUSGS() {
+  
+  var watermarkHeight = 100,
+      watermarkWidth = 200;
+  var bufferFromEdge = 25;
+  
+  // increase minimum so that you can actually see it
+  function precisionRound(number, precision) {
+    var factor = Math.pow(10, precision);
+    return Math.round(number * factor) / factor;
+  }
```

Comment:
I think you're safe to delete lines 7-11. That function was used in legend to round the number for the size of the circles.

Reply:
whoops